apps/api/src/food-index/ranking/local-popularity.ts

```diff
@@ -32,6 +32,7 @@ export async function getLocalPopularityRanking(
 
     rows.forEach((row) => {
       ranking[row.foodCode] = row.occurrences;
+      ranking[row.foodCode] = row.occurrences * row.multiplier;
```

Comment:
@mucc001 This ranking function caches popularity counter results from the database, you need to keep that in mind when testing (e.g. set stdTTL to 0 here to see changes reflected immediately)